### PR TITLE
First selenium test

### PR DESCRIPTION
**Pull request by kirilstrax**
This is a very simple, but working selenium test case

What it does
------------------
- opens WebUI, 
- navigates to Actions tab, 
- selects "core.local" and specifies the command 'uname'
- navigates to History tab
- waits for command to complete 
- checks that the output is as expected ('linux')

Limitations
-----------------
- Not using any libraries yet
- Not in pack or workflow format yet
- Although I've made it reasonably readable, it's far from ideal
- Some hardcoded values/assumptions made in terms of page structure

### CODE REVIEW

**Comment by dzimine** [comment]
@kirilstrax we need to find a better place for it, st2 is not a good one, and I don't know the answer yet, let's discuss on slack. Obvious options are 1) with st2web and 2) separate repo for e2e tests cc @DoriftoShoes

**Comment by kirilstrax** [comment]
@dzimine I submitted it here as a trial. I think separate repo is more practical, but don't have permissions to create one.
How about st2e2e? Or whatever you prefer really

File: st2tests/tests/selenium/selenium_sanity.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import datetime
```
Comment (lakshmi-kannan) [inline st2tests/tests/selenium/selenium_sanity.py]: Needs LICENSE info.

**Comment by lakshmi-kannan** [comment]
If it's in st2web, it should be all javascript right? These tests should be changed to run as part of a test suite/test runner. Our testing story cannot be here run these scripts in order. This definitely needs a discussion.

File: st2tests/tests/selenium/selenium_sanity.py (hunk 1 above)
Comment (kirilstrax) [inline st2tests/tests/selenium/selenium_sanity.py]: Do you mean for Selenium?
It's Apache 2.0 (http://www.seleniumhq.org/about/license.jsp)

**Comment by kirilstrax** [comment]
@lakshmi-kannan This could be another argument for putting selenium tests on a separate repository. Although moving them to JS driver (Node based) is possible as well.

**Comment by dzimine** [comment]
Between two Kirills, please decide what's the best. 
If you decide that selenium is WebUI tests, it may be reasonalbe to keep it in st2web with the web.
Making selenium for complete e2e tests - more reasons to move it out to a separate repo.
On languages, it would be good if we stay with JavaScript and Python.

**Comment by enykeev** [comment]
I'm pretty sure we would need several types of selenium tests to cover anything that should be covered. This one seems like a basic e2e covering st2web, st2 and potentially st2contrib. The tests may fail due to changes in any of the projects so it seems logical to me
- keep them separate (+1 for st2e2e, https://github.com/StackStorm/st2e2e here you go)
- write them in python (most of our codebase written in Python, anyone in the team can write in it so anyone would be able to fix them or spot the problem. We can formulate some kind of guidelines st2web should follow to avoid breaking tests too often)
- expect them to fail sometime (having multiple repos means there's always a window of opportunity for code and tests end up being out of sync. We definitely should not run them on repo push\merge)

We would also need selenium units and e2e specifically for st2web covering every feature we have there. Those should be in JS, take into account angular specific, be a part of st2web repository and run on every st2web build (we'll see whether it would be possible to run them for every commit in PR as the ones Travis runs for us right now.).

**Comment by Kami** [comment]
Closing since this is not relevant anymore. Tests have moved to the st2tests repo - https://github.com/StackStorm/st2tests.

**Pull request closed without merging**